src/js/setUpSite.js

```diff
@@ -65,18 +72,20 @@ const createCityMarkers = (map, data) =>
     acc[cityState] = marker;
 
     marker.bindTooltip(cityState);
-    marker.addTo(map);
+    marker.addTo(markerGroup);
     return acc;
   }, {});
 
 const setUpSite = async () => {
   setUpIcons();
   setUpAbout();
   const map = createMap();
+  const markerGroup = createMarkerGroup(map);
   addLegend(map, SCOPE_TO_COLOR);
 
   const data = await readData();
-  const citiesToMarkers = createCityMarkers(map, data);
+  const citiesToMarkers = createCityMarkers(data, markerGroup);
+  setUpDetails(markerGroup, data);
   setUpSearch(map, citiesToMarkers);
```

Comment:
This should probably be refactored to add markers to the markerGroup rather than the map directly.

Reply:
I think it already is!

Reply:
I mean `setUpSearch` in particular. You're still passing the `map` to it